docs/develop.md

```diff
@@ -0,0 +1,65 @@
+# development
+
+If you like Tigre purpose and would like to contribute this page can help you.
+
+Tigre support only python3.7+ and you need version 3.7+ to develop. We recommend that you use `Pyenv` to manage many python enviroments. Basic instructions can be found below.
+
+For develop we recommend `Poetry` and configuration can be found below too.
+
+To manipulate remote webdrivers, we has a `docker_test_image` on own reporsitory, but we recommends `Selenoid` containers. Are more stable
```

Comment:
`on own reporsitory` -> `on this repository`

`we recommends` -> `we recommend`

`Are more stable` -> `They are more stable.`

Reply:
We has a docker_test_image => we have